specification/recoveryservicesdatareplication/resource-manager/Microsoft.DataReplication/preview/2021-02-16-preview/examples/DraOperationStatus_Get.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "Gets the Dra operation status.",
```

Comment:
In docs, we should not use 'Dra' which is not a word or well-known (to general programmer audience) abbreviation. You should fix up all the docs (descriptions, titles, example names etc) at the least - as it should not break any of your investments so far.

Reply:
I will address this in the next iteration of push. Meanwhile please keep reviewing so I can address further feedback all together.